<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Courswork</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <widget class="QWidget" name="tab_main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>0</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_username">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/user.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>User Name</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_accounts">
       <attribute name="title">
        <string>Accounts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::MinimumExpanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_username_2">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../Image/user.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTabWidget" name="tabWidget_Accounts">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <widget class="QLabel" name="label_12">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Account:</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>0</y>
                <width>161</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_Accounts_add">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>210</y>
                <width>80</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_Accounts_delete">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>210</y>
                <width>80</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <widget class="QPushButton" name="pushButton_Accounts_create">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>210</y>
                <width>80</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Accounts</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView_accounts"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_payments">
       <attribute name="title">
        <string>Payments</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_username_3">
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../Image/user.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>User Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QTabWidget" name="tabWidget_Accounts_2">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <widget class="QLabel" name="label_13">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>71</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>Account:</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox_Payments">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>0</y>
                <width>161</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_Payments_add">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>210</y>
                <width>80</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
             <widget class="QPushButton" name="pushButton_Payments_create">
              <property name="geometry">
               <rect>
                <x>40</x>
                <y>210</y>
                <width>80</width>
                <height>24</height>
               </rect>
              </property>
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Payments</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView_payments"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_user">
       <attribute name="title">
        <string>User</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTabWidget" name="tabWidget_user">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="user_tab_singIn">
             <attribute name="title">
              <string>Sing in</string>
             </attribute>
             <widget class="QWidget" name="">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>40</y>
                <width>401</width>
                <height>271</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>SING IN</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Input name</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_SingIn_name"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                     <item>
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Passwor name</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLineEdit" name="lineEdit_SingIn_password"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_SingIn_SIngIn">
                   <property name="text">
                    <string>Sing in</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>No account?</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_SingIn_SIngUp">
                   <property name="text">
                    <string>Sing up</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
            <widget class="QWidget" name="user_tab_singUP">
             <attribute name="title">
              <string>SIng up</string>
             </attribute>
             <widget class="QLabel" name="label_10">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>30</y>
                <width>311</width>
                <height>41</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>SING IN</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_SingUp_SIngIn_Up">
              <property name="geometry">
               <rect>
                <x>70</x>
                <y>160</y>
                <width>251</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Sing up</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="user_tab_displayedUser">
             <attribute name="title">
              <string>display info</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
